flexmeasures/data/scripts/clean_database.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+MAIN_DIR=$(pwd)
+
+# function for checking database existence
+function is_database() {
+ cd /tmp
```

Comment:
This is now the only place in the script where we go to /tmp. Why is it not necessary in create_database and/or delete_database?

Reply:
well, I didn't get any directory permission issue if I don't go to /tmp in these two function. So that's I didn't use it there.

Reply:
If i use `-i` with `sudo` then I don't need to change directory